subsys/net/lib/nrf_cloud/coap/cddl/nrf_cloud_coap_agnss.cddl

```diff
@@ -28,11 +25,8 @@ filtered = 3
 mask = 4
 mcc = 5
 mnc = 6
-requestType = 7
-rsrp = 8
-tac = 9
-rtAssistance = 10
-custom = 11
+rsrp = 7
```

Comment:
This changes the IDs for `rsrp` and `tac`. Does this mean that there has been backwards incompatible changes to the cloud API?

Reply:
Numbers were changed only on CoAP's agnss endpoint, not for the existing apgs endpoint, so older code should continue to work.